docs/src/GettingStarted/RunningClimateMachine.md

```diff
@@ -28,27 +28,42 @@ $ julia --project experiments/AtmosGCM/heldsuarez.jl
 ## Input and output
 
 The `ClimateMachine` provides the [`ArtifactWrappers`](@ref
-ArtifactWrappers-docs) module to assist a driver in sourcing input data
-for a simulation, but any mechanism may be used.
+ArtifactWrappers-docs) module to assist a driver in sourcing input data for a
+simulation, but any mechanism may be used.
 
 Output takes the form of various [groups of diagnostic variables](@ref
-Diagnostics) that are written to NetCDF files at specified intervals
-by the `ClimateMachine` when configured to do so by a driver. A
-driver may alternatively (or additionally) configure output of the
-conservative and auxiliary state variables to VTK files. However,
-the default behavior of the `ClimateMachine` is to not generate
-any output, irrespective of driver setup. The user must explicitly
-enable output by using the appropriate [command line argument](@ref
-ClimateMachine-args) (e.g. `--diagnostics default`) or by setting
-the appropriate [environment variable](@ref ClimateMachine-envvars)
-(e.g. `CLIMATEMACHINE_SETTINGS_DIAGNOSTICS=default`).
-
-## [Command line arguments](@id ClimateMachine-args)
-
-The `ClimateMachine` allows some aspects of its behavior to be controlled
-by command line arguments. Many drivers make use of this facility
-(but some do not). The command line options recognized can be examined
-interactively, for example:
+Diagnostics-vars) that are written to NetCDF files at specified intervals by
+the `ClimateMachine` when configured to do so by a driver.
+
+The `ClimateMachine` can also output conservative and auxiliary state variables
+to VTK files at specified intervals.
+
+Whether or not output is generated, and if so, at what interval, is a
+`ClimateMachine` _setting_.
+
+## `ClimateMachine` settings
+
+Some aspects of the `ClimateMachine`'s behavior can be controlled via its
+_settings_ such as use of the GPU, diagnostics output and frequency,
+checkpointing/restarting, etc. There are 3 ways in which these settings can be
+changed:
+
+1. [Command line arguments](@ref ClimateMachine-clargs) have the highest
+   precedence, but it is possible for a driver to disable parsing of command
+   line arguments. In such a case, only the next two ways can be used to change
+   settings.
+
+2. [Environment variables](@ref ClimateMachine-envvars) are considered after
+   command line arguments.
+
+3. [Programmatic settings](@ref ClimateMachine-kwargs) have the lowest
+   precedence.
+
+### [Command line arguments](@id ClimateMachine-clargs)
+
+If a driver configures the `ClimateMachine` to parse command line
+arguments (most do), you can query the list of arguments understood,
```

Comment:
```suggestion
arguments by setting `parse_clargs = true` in `ClimateMachine.init()`, you can query the list of arguments understood,
```
I think there should be some indication for developers on how to turn this flag on.